code/go/0chain.net/chaincore/chain/config.go

```diff
@@ -530,7 +530,10 @@ func (c *ConfigImpl) FromViper() error {
 	conf.DbsEvents.MaxIdleConns = viper.GetInt("server_chain.dbs.events.max_idle_conns")
 	conf.DbsEvents.MaxOpenConns = viper.GetInt("server_chain.dbs.events.max_open_conns")
 	conf.DbsEvents.ConnMaxLifetime = viper.GetDuration("server_chain.dbs.events.conn_max_lifetime")
+
+	conf.DbsEvents.Debug = viper.GetBool("server_chain.dbs.events.debug")
 	conf.DbsEvents.AggregatePeriod = viper.GetInt64("server_chain.dbs.events.aggregate_period")
+	conf.DbsEvents.Debug = viper.GetBool("server_chain.dbs.events.page_limit")
```

Comment:
PageLimit?

Reply:
https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/dbs/event/enums.go#L10
```go
const pageLimit = int64(50)
```
Maybe it need not be an event database setting, but handlers need to pick it up from somewhere.

Reply:
I mean shouldn't it be assigned to `conf.DbsEvent.PageLimit` instead of `conf.DbsEvents.Debug`